openquake/server/utils.py

```diff
@@ -11,11 +11,17 @@ def getusername(request):
     compatibility.
     """
 
```

Comment:
I would change the name from get_username to get_user since now it returns more than a name